Yafc/Windows/PreferencesScreen.cs

```diff
@@ -89,11 +89,11 @@ public class PreferencesScreen : PseudoScreen {
         /// <summary>Add a GUI element that opens a popup to allow the user to choose from the <paramref name="list"/>, which triggers <paramref name="select"/>.</summary>
         /// <param name="text">Label to show.</param>
         /// <param name="width">Width of the popup. Make sure it is wide enough to fit text!</param>
-        private void ChoiceObject<T>(ImGui gui, string text, T[] list, T current, Action<T> select, float width = 20f) where T : FactorioObject {
+        private void ChoiceObject<T>(ImGui gui, string text, T[] list, T current, Action<T> select, float width = 20f, bool allowNone = false) where T : FactorioObject {
```

Comment:
We are trying to improve the documentation little by little (no one volunteers to update all in one go :wink: ), so could you please document the new parameter you added to this function as another little step?